### DISCUSSION TITLE
[5.0x] FR: Make provisional drafts of nested entries discoverable (cards)

### DISCUSSION THREAD

**Discussion by wsydney76** [craftcms/cms#14604] [Ideas] [answered]
We have a matrix field with view mode 'Cards' like this:

![image](https://github.com/craftcms/cms/assets/60010597/9093be20-3bab-4529-8cfb-61d9f43eef6e)

As it works now, editing a nested entry in a slideout creates a new provisional draft of this nested entry. 

However, the slideout can be closed without saving (just click outside), leaving no visual hint for the editor. 

So there should be some indicator in the card (icon, text, blue left border, whatever) that an open edit exists.

**Comment by brandonkelly** [comment]
Agree we should do this, especially alongside #14603 when the provisional draft changes will end up getting published on parent save.

**Comment by brandonkelly** [comment, marked as answer]
This was resolved in Craft 5.2 via #14975.
